src/trapfunc.cpp

```diff
@@ -1573,17 +1573,6 @@ bool trapfunc::cast_spell( const tripoint_bub_ms &p, Creature *critter, item * )
         if( !tr.has_flag( json_flag_UNCONSUMED ) ) {
             here.remove_trap( p );
         }
-        if( tr.has_flag( json_flag_PROXIMITY ) ) {
```

Comment:
Is this flag completely unused?

Reply:
it is used in map::creature_on_trap(), to populate tr_proximity, that stores all traps in proximity, and potentially trigger them